tests/Benchmarks/Program.cs

```diff
@@ -17,9 +18,17 @@ public class Program
         /// <param name="args"></param>
         public static void Main(string[] args)
             => BenchmarkSwitcher
-                .FromAssembly(typeof(Program).Assembly)         
+                .FromAssemblyAndTypes(typeof(Program).Assembly, GetGenericBenchmarks())         
                 .Run(args/*, CreateClrVsCoreConfig() uncomment it to run Clr vs .NET Core comparison*/);
 
+        // BenchmarkDotNet can run generic benchmarks, if it knows what generic type arguments to use
+        private static Type[] GetGenericBenchmarks()
+            => new[]
+            {
+                typeof(HttpParser<Request>),
```

Comment:
How do we extend this when new tests are added with different generic types?

Reply:
In that case, we will need to specify the generic arguments like in this example.
Why is that? When BDN finds a generic type (let's say `Benchmark<T>`) it does not know which `T` should be used. So this has to be specified in an explicit way.

I just created an issue for that in BDN to allow doing this by using attributes: https://github.com/dotnet/BenchmarkDotNet/issues/745